pynecone/constants.py

```diff
@@ -195,11 +195,10 @@ def get_url(self) -> str:
 class PathArgType(SimpleNamespace):
     """Type of pathArg extracted from URI path."""
 
-    SINGLE = str("arg_single")
-    LIST = str("arg_list")
+    SINGLE = "arg_single"
```

Comment:
I believe this was a hack needed for dynamic routing by @Lendemor in his PR. Can we revert this and add a comment above stating that this is a needed workaround